ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--
+
+Have you read the Contribution Guide?
+By filing an Issue, you are expected to comply with it: https://github.com/modelica/fmi-standard/blob/master/CONTRIBUTING.adoc
+
+Do you want to ask a question? Are you looking for support? 
```

Comment:
Add "This issue tracker is used for the development of the FMI Standard." ? ....